e2e-tests/e2e-fixtures/constants.fixture.js

```diff
@@ -33,6 +33,7 @@ export const DEAL_STATUS = {
   // statuses specific to the entire deal
   READY_FOR_APPROVAL: "Ready for Checker's approval",
   CHANGES_REQUIRED: "Further Maker's input required",
+  CANCELLED: 'Cancelled',
```

Comment:
Can this just be imported from libs/common?

Reply:
Good plan, now doing